src/assets/css/components/battle.css

```diff
@@ -13,6 +13,16 @@
   border-radius: 20px;
 }
 
+.battle__overlay {
+  position: absolute;
+  top: 0px;
+  left: 0px;
+  width: 100vw;
+  height: 100vh;
+  background: rgba(0, 0, 0, 0.7);
```

Comment:
maybe chabge 0.7 to 0.1, I think it looks better and player still cannot touch anything